rpcs3/Emu/RSX/Common/TextureUtils.cpp

```diff
@@ -457,18 +457,36 @@ std::vector<rsx_subresource_layout> get_subresources_layout_impl(const RsxTextur
 
 	std::tie(h, depth, layer) = get_height_depth_layer(texture);
 
-	int format = texture.format() & ~(CELL_GCM_TEXTURE_LN | CELL_GCM_TEXTURE_UN);
+	const auto format = texture.format() & ~(CELL_GCM_TEXTURE_LN | CELL_GCM_TEXTURE_UN);
+	const auto pitch = texture.pitch();
 
 	const u32 texaddr = rsx::get_address(texture.offset(), texture.location(), HERE);
 	auto pixels = vm::_ptr<const std::byte>(texaddr);
 
 	const bool is_swizzled = !(texture.format() & CELL_GCM_TEXTURE_LN);
 	const bool has_border = !texture.border_type();
 
+	if (!is_swizzled)
+	{
+		if (pitch) [[likely]]
+		{
+			if (const auto packed_pitch = get_format_packed_pitch(format, w, has_border, false);
```

Comment:
isn't this variable unused? may just say
pitch < get_format_packed_pitch(format, w, has_border, false)